content/docs/command-reference/metrics/index.md

```diff
@@ -1,47 +1,104 @@
     diff                Show changes in metrics between commits
 ```
 
 ## Description
 
-In order to track metrics associated to machine learning experiments, DVC has
-the ability to mark a certain stage <abbr>outputs</abbr> as files containing
-metrics to track (see the `--metrics` option of `dvc run`). Metrics are
-project-specific floating-point values e.g. `AUC`, `ROC`, etc.
+In order to track the basic performance of machine learning experiments, DVC has
+the ability to mark a certain stage <abbr>outputs</abbr> _scalar metrics_ to
+track.
 
-Supported file formats: JSON. Metrics can be organized in a tree hierarchy in a
-JSON file. DVC addresses the metrics by the tree path.
+These metrics are project-specific floating-point or integer values e.g. AUC,
+ROC, _false positives_, etc. They can be defined with the `-m` (`--metrics`) and
+`-M` (`--metrics-no-cache`) options of `dvc run`.
 
-DVC itself does not ascribe any specific meaning for these numbers. Usually
-these numbers are produced by the model training or model evaluation code and
-serve as a way to compare and pick the best performing experiment.
+### Types of metrics
 
-[Add](/doc/command-reference/metrics/add),
-[show](/doc/command-reference/metrics/show),
-[diff](/doc/command-reference/metrics/diff),
-[modify](/doc/command-reference/metrics/modify), and
-[remove](/doc/command-reference/metrics/remove) commands are available to set up
-and manage <abbr>DVC project</abbr> metrics.
+DVC has two concepts for metrics, that represent different results of machine
+learning training or data processing:
+
+1. `dvc metrics` represent **scalar numbers** such as AUC, _true positive rate_,
+   etc.
+2. `dvc plots` can be used to visualize **continuous metrics** such as AUC
+   curves, loss functions, confusion matrices, etc.
+
+In contrast to continuous metrics,
+[scalar metrics](/doc/command-reference/metrics) should be stored in a
+hierarchical files. Unlike its `dvc plots` counterpart, `dvc metrics diff` can
+report the numeric difference between the metrics in different experiments, for
+example an `AUC` metrics that is `0.801807` and gets increase by `+0.037826`:
+
+```dvc
+$ dvc metrics diff
+    Path       Metric    Value      Change
+summary.json   AUC      0.801807   0.037826
+```
+
+### Supported file formats
+
+Metrics can be organized as tree hierarchies in JSON or YAML files. DVC
+addresses specific metrics by the tree path. In the JSON example below, five
+metrics are presented: `train.accuracy`, `train.loss`, `train.TN`, `train.FP`
+and `time_real`.
+
+```json
+{
+  "train": {
+    "accuracy": 0.9886999726295471,
+    "loss": 0.041855331510305405,
+    "TN": 473,
+    "FP": 845
+  },
+  "time_real": 344.61309599876404
+}
+```
+
+DVC itself does not ascribe any specific meaning for these numbers. Usually they
+are produced by the model training or model evaluation code and serve as a way
+to compare and pick the best performing experiment.
+
+### Default metric files
```

Comment:
should become part of the description

Reply:
do we need a similar paragraph in plots?

Reply:
It is part of the description.

Reply:
> do we need a similar paragraph in plots?

No, plots require specifying a target/datafile

Reply:
p.s. this led me to discover the following issues: #3925 and #3924

Reply:
I mean iterative/dvc/issues/3925 and iterative/dvc/issues/3924

Reply:
it overlaps a lot:

For example:

> Metric files can be added to `dvc.yaml` with the `--metrics` (`-m`) or
`--metrics-no-cache` (`-M`) options of `dvc run`, or manually to the `metrics`
section of a stage in `dvc.yaml`:

and

> This kind of metrics can be defined with the `-m` (`--metrics`) and `-M`
(`--metrics-no-cache`) options of `dvc run`.

Why can't we make it a single paragraph?

anyway, probably not a big deal .. we can keep it like this for now

Reply:
> > do we need a similar paragraph in plots?
>
> No, plots require specifying a target/datafile

Actually probably we'll need it but this will be done as a result of resolving iterative/dvc/issues/3925 (I'll keep track).

> it overlaps a lot:

Yeah I didn't notice before. Removed redundancy in 47ad3116 (part of #1382)!

> Why can't we make it a single paragraph?

I could but would probably want to completely remove the dvc.yaml example in that case and the `cache:false` note. Also not sure it makes sense to have this info before the **Supported file formats** section. No strong opinion, I'll do it if you think it's best. Please lmk